deploy/charts/arcadia/values.yaml

```diff
@@ -122,17 +122,21 @@ dataprocess:
   enabled: true
   image: kubeagi/data-processing:v0.1.0
   port: 28888
-  env:
-    minio:
-      secure: "False"
-      apiURL: arcadia-minio:9000
-    knowledge:
-      chunkSize: 500
-      chunkOverlap: 50
-    postgres:
```

Comment:
Since our llm application also requires this postgresql, I move this out of `dataprocessing`